README.md

```diff
@@ -869,6 +869,10 @@ DESC       LOW          HIGH          SIZE ATTR  ID TASK
 we can see from the `map` output has been sized to only 256 bytes.)
 
 
+### `humility monorail`
+
+No documentation yet for `humility monorail`; pull requests welcome!
```

Comment:
At some point -- though perhaps not with this commit -- it would be great to get docs on `humliity monorail`...

Reply:
Yeah, dropped some docs so that the CLI output isn't empty